geode-lucene/src/main/java/org/apache/geode/cache/lucene/internal/repository/IndexRepositoryImpl.java

```diff
@@ -54,19 +55,20 @@
   private DocumentCountSupplier documentCountSupplier;
   private final DistributedLockService lockService;
   private String lockName;
+  private LuceneIndex index;
 
   private static final Logger logger = LogService.getLogger();
 
   // For test purposes
   IndexRepositoryImpl(Region<?, ?> region, IndexWriter writer, LuceneSerializer serializer,
       LuceneIndexStats stats, Region<?, ?> userRegion) throws IOException {
     this(region, writer, serializer, stats, userRegion,
-        ((DistributedRegion) region).getLockService(), "NoLockFile");
+        ((DistributedRegion) region).getLockService(), "NoLockFile", null);
```

Comment:
I'm not sure why you are passing null here, but maybe we should just get rid of this constructor?

Reply:
yes, I still remembered to do that yesterday, but just forgot